### PR TITLE
feat: Updates to releases progress bar

### PR DESCRIPTION
**Pull request by denamwangi**
This adds:
- Dynamicness to the bar depending on how many steps are complete
- A single cta for what needs to be done next
- A statement on why they should even bother setting it up

At it's longest:
screen shot 2018-11-07 at 4 30 28 pm

To Do:
- [x] fix padding :woman_facepalming:
- [x] add specific url for each step

### CODE REVIEW

File: src/sentry/static/sentry/app/views/projectReleases/releaseProgress.jsx [hunk 1]
```diff
@@ -44,6 +45,11 @@ class ReleaseProgress extends AsyncComponent {
     } else if (stateKey === 'setupStatus') {
       this.getRemainingSteps(data);
     }
+    let {organization, project} = this.context;
+    analytics('releases.progress_bar_viewed', {
+      org_id: parseInt(organization.id, 10),
+      project_id: parseInt(project.id, 10),
```
Comment (adhiraj) [inline src/sentry/static/sentry/app/views/projectReleases/releaseProgress.jsx]: can we record how complete it was when they saw it?

File: src/sentry/static/sentry/app/views/projectReleases/releaseProgress.jsx [hunk 2]
```diff
@@ -3,16 +3,34 @@ import React from 'react';
...
 
 const STEPS = {
-  tag: t('Tag an error'),
-  repo: t('Link to a repo'),
-  commit: t('Associate commits'),
-  deploy: t('Tell sentry about a deploy'),
+  tag: {
+    desc: t('Tag an error'),
+    url: 'tag-errors',
+    msg: 'knowing which errors were introduced in a release, ',
```
Comment (denamwangi) [inline src/sentry/static/sentry/app/views/projectReleases/releaseProgress.jsx]: I can work on making this bold in particular spots to highlight bits of each but this works for now

Comment (adhiraj) [reply src/sentry/static/sentry/app/views/projectReleases/releaseProgress.jsx]: not too important

File: src/sentry/static/sentry/app/views/projectReleases/releaseProgress.jsx [hunk 3]
```diff
@@ -45,15 +63,40 @@ class ReleaseProgress extends AsyncComponent {
     }
   }
 
+  buildMessage(remainingSteps) {
+    let keys = remainingSteps.map(step => step.step);
+    let message = 'Save time by ';
```
Comment (adhiraj) [inline src/sentry/static/sentry/app/views/projectReleases/releaseProgress.jsx]: We can try "You're missing out on "

**Review by adhiraj — APPROVED** [review]

**Pull request merged**